config/config.go

```diff
@@ -1506,3 +1521,67 @@ func isValidCookieSize(maxCookieSize int) error {
 	}
 	return nil
 }
+
+type TmaxAdjustments struct {
+	// Flag indicating whether the tmax feature is enabled or not
+	Enabled bool `mapstructure:"enabled"`
+	// The maximum allowable tmax for the auction endpoint
+	// The tmax value used for the auction endpoint will be the minimum of the auction_max and tmax specified in the incoming request
+	AuctionMax int `mapstructure:"auction_max"`
+	// The maximum allowable tmax for the video endpoint
+	// The tmax value used for the video endpoint will be the minimum of the video_max and tmax specified in the incoming request
+	VideoMax int `mapstructure:"video_max"`
+	// The maximum allowable tmax for the AMP endpoint
+	// The tmax value used for the AMP endpoint will be the minimum of the amp_max and tmax specified in the incoming request
+	AmpMax int `mapstructure:"amp_max"`
+	// The minimum duration needed for a bidder to respond back
+	// PBS will not send an HTTP request to the bidder server, if the time needed for PBS processing, adapter's MakeRequests() implementation, building Prebid headers, and applying GZip compression (if needed) is less than bidder_response_min.
+	BidderResponseMin int `mapstructure:"bidder_response_min"`
```

Comment:
Consider changing the type for `BidderResponseMin`, `BidderLatencyAdjustment` and, `UpstreamResponseDuration` to be unsigned integers as we only ever expect positive values for these fields.

Reply:
makes sense to change make them unsigned integers. https://github.com/prebid/prebid-server/pull/2723/commits/8caafd3c2661aea7b7d3151320b6350f175bdb28